spinalcordtoolbox/vertebrae/detect_c2.py

```diff
@@ -0,0 +1,149 @@
+import sys
```

Comment:
CLIs go under scripts/

Reply:
you could create a CLI for `sct_detect` (just an example), that would call the API detect_c2 (or detect_object, to be more generic). This API would point to an object-detection model. Here, it is C2-C3 disc, but it could be something else.
Since this is out of the scope of this PR, open an issue to propose this change. When opening the issue, mention that `sct_detect_pmj` could fall into this more generic new CLI